### PR TITLE
Skupack support for dell servers

### PR DESCRIPTION
**Pull request by cgx027**
Servers including R630, R730 and C6320

This PR depends on 
https://github.com/RackHD/on-taskgraph/pull/122 and https://github.com/RackHD/on-tasks/pull/239

### CODE REVIEW

File: dell-c6320/workflows/dell-discovery-graph.json [hunk 1]
```diff
@@ -0,0 +1,39 @@
+{
+    "friendlyName": "Dell Discovery",
+    "injectableName": "Graph.Dell.Discovery",
+    "options": {
+        "bootstrap-ubuntu": {
+            "overlayfs": "common/dell.raid.overlay.cpio.gz"
+        },
+        "create-raid": {
+            "path": "/opt/MegaRAID/perccli/perccli64"
+        }
```
Comment (iceiilin) [inline dell-c6320/workflows/dell-discovery-graph.json:7]: Will create-raid task be used here?

**Comment by iceiilin** [comment]
After discussed with @pengz1 , dell-perccli-megaraid-catalog.json is used by user to manually run the perccli catalog. After adding sku specific graph in this PR, where the perccli catalog will be fetched automatically, dell-perccli-megaraid-catalog.json can be removed.

**Comment by iceiilin** [comment]
From @pengz1 , dell-perccli-megaraid-catalog.json can still be kept in case user modify the raid and want to refresh the catalog themselves.

:+1: after address the other comment.

File: dell-c6320/workflows/dell-discovery-graph.json (hunk 1 above)
Comment (pengz1) [inline dell-c6320/workflows/dell-discovery-graph.json:7]: “Create-raid” option should be deleted

File: dell-c6320/workflows/dell-discovery-graph.json [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+                "bootstrap-ubuntu": "succeeded"
+            }
+        },
+        {
+            "label": "dell-racadm-get-config-catalog",
+            "taskName": "Task.Dell.Racadm.GetConfigCatalog",
+            "waitOn": {
+                "catalog-perccli": "succeeded"
+            }
+        },
```
Comment (pengz1) [inline dell-c6320/workflows/dell-discovery-graph.json]: Without OBM setting, this tasks will fail.

**Comment by cgx027** [comment]
Thanks @pengz1  and @iceiilin , your concerns are address in my last two commits.

**Comment by pengz1** [comment]
+1

**Comment by anhou** [comment]
:+1:

**Pull request merged**